test.sh

```diff
@@ -65,6 +65,10 @@ function docker_compose_run {
     invoke_docker_compose run --rm --user `id -u`:`id -g` "$@"
 }
 
+function docker_compose_run_fe {
+  invoke_docker_compose run --rm frontend_tester npm run "$@"
+}
+
```

Comment:
This and the rest of the changes for this file can probably be removed after updating from master?

Reply:
Done.